README.md

```diff
@@ -1,49 +1,47 @@
 Deploy new MySQL database using our playbooks
 =============================================
 
-This ansible repository deploys mysql server on OpenStack provider. 
+This ansible repository deploys mysql server on OpenStack provider.
 
-Create a OpenStack environment
-------------------------------
+Create an OpenStack environment
+-------------------------------
 
-1. Create volume for data, as big as you need. It should be empty.
-2. Create a security group for database servers it should have following rules:
-    1. Incoming traffic is allowed only for port 22
-    2. Outgoing traffic should be allowed. 
+1. Create a volume for data, as big as you need. It should be empty.
+2. Create a security group for database servers. It should have following rules:
+    1. Incoming traffic is allowed only for port 22.
+    2. Outgoing traffic should be allowed.
     3. When you provision servers using the database allow them using their IP.
-3. Create instance using Ubuntu 14.04 image. We suggest you use vanilla image from 
-   [ubuntu page](https://cloud-images.ubuntu.com/). This image might be small, mysql 
+3. Create an instance using an appropriate image, such as Ubuntu 16.04. We suggest you use vanilla image from
+   [ubuntu page](https://cloud-images.ubuntu.com/). This image might be small; MySQL
    data will be stored elsewhere.
-4. Attach the data volume to the vm (go to `Volumes` -> your data volume -> this down `v` arrow near Edit Volume -> 
-   `Edit Attachments`).    
-5. Manual step: login to administrative user of the database machine and format /dev/vdb using ext4. 
-   To do this login to the instance and issue `fdisk /dev/vdb` and then issue:    
+4. Attach the data volume that you created earlier to the VM (go to `Volumes` -> <data volume> -> expand dropdown next to "Edit Volume" ->
```

Comment:
In the [rich diff view](https://github.com/open-craft/deployment-deploy-databases/pull/9/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8), this "<data volume>" is being hidden. It also is being hidden in the [actual rendered page](https://github.com/open-craft/deployment-deploy-databases/blob/e38c3238d794205a7c2538cb91b0a32ea1dbbb55/README.md#create-an-openstack-environment).

Reply:
@UmanShahzad Thanks for catching that! Fixed.